setup/sale_triple_discount/setup.py

```diff
@@ -2,5 +2,9 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
```

Comment:
@legalsylvain @grindtildeath Ok, I've found a solution to limit the version of account module here.

I'll merge to fix CI builds in v16

Reply:
Nice !